### PR TITLE
RHPAM-2040: Custom coloring - Unexpected error in instance details af…

### PR DESCRIPTION
**Pull request by nmirasch**
…ter process start
added control for empty values passed to getProcessInstanceImageCustomColor as customized colors

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @mswiderski @MarianMacik @jstastny-cz  I just added the control for empty values at this call. That situation  have been found by automatic testing that it seems don't have time for load user preferences properly and is passing empty values. I think this control here makes stronger the service point. Please take a look and let me know what you think. Thanks!

File: kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UIServicesClientImpl.java [hunk 1]
```diff
@@ -210,6 +212,21 @@ public String getProcessInstanceImage(String containerId, Long processInstanceId
         }
     }
 
+    protected String createColorURLParams(String completeNodeColor, String completeNodeBorderColor, String activeNodeBorderColor){
```
Comment (jstastny-cz) [inline kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UIServicesClientImpl.java]: please resolve white-space issues in the newly added code (missing space between ')' and '{', missing space around '=', no spacing around '!=', ...), reformat according to code style.

File: kie-server-parent/kie-server-remote/kie-server-client/src/test/java/org/kie/server/client/impl/UIServicesClientImplTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import org.kie.server.api.rest.RestURI;
+import org.kie.server.client.KieServicesConfiguration;
+import org.kie.server.client.KieServicesFactory;
+
+import static org.junit.Assert.assertEquals;
+
+public class UIServicesClientImplTest {
+
+    @Test
+    public void testCreateColorUrlParams() {
```
Comment (jstastny-cz) [inline kie-server-parent/kie-server-remote/kie-server-client/src/test/java/org/kie/server/client/impl/UIServicesClientImplTest.java]: reformat the method, white-space issues

File: kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UIServicesClientImpl.java (hunk 1 above)
Comment (nmirasch) [reply kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UIServicesClientImpl.java]: @jstastny-cz updated

File: kie-server-parent/kie-server-remote/kie-server-client/src/test/java/org/kie/server/client/impl/UIServicesClientImplTest.java (hunk 2 above)
Comment (nmirasch) [reply kie-server-parent/kie-server-remote/kie-server-client/src/test/java/org/kie/server/client/impl/UIServicesClientImplTest.java]: @jstastny-cz done

**Comment by nmirasch** [comment]
@mswiderski regarding this PR has been approved for a while, could it be merged? it's just to close the related jira. Thanks!

**Pull request merged**